src/coprocessor/select/topn_heap.rs

```diff
@@ -174,6 +174,8 @@ impl PartialOrd for SortRow {
     }
 }
 
+unsafe impl Send for TopNHeap {}
```

Comment:
I think you can change `Arc<RefCell<>>` to `Rc<RefCell<>>`.

Reply:
`Rc<RefCell<_>>` is `!Send`.

Reply:
maybe do it in another little PR is better?